server/server.py

```diff
@@ -254,6 +262,30 @@ async def details(request: Request):
          "args": args.__dict__}
     )
 
+def worker(sentence, speaker_id):
+
+    print(" > Model input: {}".format(sentence))
+    print(" > Speaker Idx: {}".format(speaker_id))
+    if use_aliases:
+        input_speaker_id = new_speaker_ids[speaker_id]
+    else:
+        input_speaker_id = speaker_id
+
+    synthesizer = Synthesizer(
+        tts_checkpoint=model_path,
+        tts_config_path=config_path,
+        tts_speakers_file=speakers_file_path,
+        tts_languages_file=None,
+        vocoder_checkpoint=vocoder_path,
+        vocoder_config=vocoder_config_path,
+        encoder_checkpoint="",
+        encoder_config="",
+        use_cuda=args.use_cuda,
+    )
```

Comment:
From what I see in the execution, worker initializes the model, each time it wants to do inference. This introduces overhead. In principle worker has to initialize the model only once in the beginning and in each inference step should just do inference taking advantage of the loaded model. I don't have a suggestion now, we talk next week.

Reply:
@mllopartbsc can you take a look at this, this week? Thanks.